src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java

```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
         if (hero == null) return emptySkillList;
 
         HeroClass heroClass = hero.getHeroClass();
-        HeroClass secondClass = hero.getSecondaryClass();
+        HeroClass secondClass = hero.getSecondaryClass();();
+        HeroClass raceClass = hero.getSecondaryClass();
         Set<String> primarySkills = new HashSet<>();
         Set<String> secondarySkills = new HashSet<>();
+        Set<String> raceSkills = new HashSet<>();
         addSkills(hero, heroClass, primarySkills, showUnuseable, showPassive);
         addSkills(hero, secondClass, secondarySkills, showUnuseable, showPassive);
+        addSkills(hero, raceClass, raceySkills, showUnuseable, showPassive);
```

Comment:
Should raceySkills be raceSkills here?

Reply:
A fat fingered typo, thanks for the review.